examples/src/main/scala/org/apache/spark/examples/snappydata/WorkingWithJson.scala

```diff
@@ -102,6 +102,38 @@ object WorkingWithJson extends SnappySQLJob {
 
     val allPersons = nameAndAddress.toJSON
     allPersons.show(truncate = false)
+
+    // Multi-line json example
+    println("Multi-line json example\n################")
+    val locomotives_path = s"${jobConfig.getString("json_resource_folder")}/locomotives.json"
+    val locomotivesDF = snSession.read.option("wholeFile", "true").json(locomotives_path)
```

Comment:
Any exception to be thrown if locomotives.json file is not present at the desired location?

Reply:
We don't want to handle the exception in a special way. It should fail, and fail fast. Moreover, since the file is part of examples, it will always be available.